### PR TITLE
Rancher 971 - Add mock-server to folio-testing-kartate ns

### PR DESCRIPTION
**Pull request by vkapylou**
[RANCHER-971](https://issues.folio.org/browse/RANCHER-971)

Successful build with changes --> https://jenkins-aws.indexdata.com/job/Rancher/job/vasili-workflow/job/RANCHER-971/74/console
Karate tests run - https://jenkins-aws.indexdata.com/job/Testing/job/Karate%20tests%20RANCHER-971/3/

### CODE REVIEW

**Review by guram-jalaghonia — APPROVED** [review]

File: pipelines/folioRancher/folioNamespaceTools/manageNamespace/createNamespaceFromBranch/Jenkinsfile [hunk 1]
```diff
@@ -13,9 +13,9 @@ import org.jenkinsci.plugins.workflow.libs.Library
 
 //TODO remove branch before merge to master
 
-@Library('pipelines-shared-library@RANCHER-741-Jenkins-Enhancements') _
+@Library('pipelines-shared-library@RANCHER-971') _
 
-CONFIG_BRANCH = 'RANCHER-741-Jenkins-Enhancements'
+CONFIG_BRANCH = 'RANCHER-971'
```
Comment (OHaimanov) [inline pipelines/folioRancher/folioNamespaceTools/manageNamespace/createNamespaceFromBranch/Jenkinsfile:18]: Please do not forget to switch back to RANCHER-741-Jenkins-Enhancements before merge

File: src/org/folio/Constants.groovy [hunk 2]
```diff
@@ -8,7 +8,7 @@ class Constants {
     static String AWS_S3_DATA_EXPORT_ID = 'ci-data-export-s3'
     static String AWS_S3_POSTGRES_BACKUPS = 'manage-postgres-db-backups-s3'
     static String AWS_EKS_VPC_NAME = 'folio-rancher-vpc'
-    static String AWS_EKS_ADMIN_USERS = 'oleksandrhaimanov,kseniiadubniak,vitaliiustalu,volodymyrkartsev,vasilikapylou,renatsafiulin,tarasspashchenko,stanislav'
+    static String AWS_EKS_ADMIN_USERS = 'oleksandrhaimanov,kseniiadubniak,volodymyrkartsev,vasilikapylou,renatsafiulin,tarasspashchenko,stanislav'
```
Comment (OHaimanov) [inline src/org/folio/Constants.groovy:11]: If you edit this var, let add Eldiar and Guram here)

File: vars/folioHelm.groovy [hunk 3]
```diff
@@ -1,6 +1,9 @@
+import hudson.AbortException
 import org.folio.Constants
 import org.folio.models.RancherNamespace
 import org.folio.utilities.Logger
+import org.folio.utilities.Tools
```
Comment (OHaimanov) [inline vars/folioHelm.groovy:5]: I think this import here not needed

**Review by OHaimanov — APPROVED** [review]

File: vars/folioHelm.groovy (hunk 3 above)
Comment (vkapylou) [reply vars/folioHelm.groovy]: Removed.

File: src/org/folio/Constants.groovy (hunk 2 above)
Comment (vkapylou) [reply src/org/folio/Constants.groovy]: Sure)

**Review by renatsafiulin — APPROVED** [review]

**Pull request merged**